pkg/security/secl/go.mod

```diff
@@ -23,6 +23,7 @@ require (
 
 require (
 	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
```

Comment:
what is this second block for?

Reply:
i'm not sure what go tidy is doing under the hood here, but since it's autogenerated and the tests pass, approving